### PR TITLE
Fix some rendering issues with the header and profile

### PR DESCRIPTION
**Pull request by anthonydillon**
Fixes: https://github.com/juju/juju-gui/issues/1612

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- I gisf check the import export buttons are aligned with the env switcher
- Click on the env switcher and check the drop down appears in the correct place
- Go to profile
- Expand and close the models and see the model names do not disappear

_I didn't remove the logic to hide the env switcher when in profile view as there must be logic there for a reason. Question for UX_

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4994/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4995/
Test PASSed.

File: jujugui/static/gui/src/app/components/user-profile/entity/entity.js [hunk 1]
```diff
@@ -254,7 +254,7 @@ YUI.add('user-profile-entity', function() {
       var isCharm = type === 'charm';
       // Model names will be in the format "username/model-name" so we have to
       // extract the part we need.
-      var name = isModel ? entity.name.split('/')[1] : entity.name;
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/entity/entity.js]: I'm not sure that this change is correct for all platforms. I'm pretty sure JEM still returns them user/model

**Comment by hatched** [comment]
This will need to be qa'd in all three deployment environments to make sure that it resolves the issues for all and doesn't introduce any side effects.

File: jujugui/static/gui/src/app/components/user-profile/entity/entity.js (hunk 1 above)
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/user-profile/entity/entity.js]: @hatched I have re-factored this to keep the split name by slash if the name contains a `/`. This raises the question of can model names contain `/` in there name?

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5000/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5001/
Test PASSed.

**Comment by kadams54** [comment]
**QA Notes:**

Running locally, against staging JEM.

* When I opened the model switcher, the diamond on the dropdown was aligned with the down arrow in the model switcher input above it. If that's the correct alignment then everything looks good. I was a little surprised because I expected the dropdown to be left-aligned with the input, but maybe that was done to avoid overlapping with the inspector?
* I clicked on the models and they expanded while retaining the name. That said, the name string shown in collapsed state did split up into its name and owner parts on expansion. Just wanted to make sure that was also intentional.

**Comment by anthonydillon** [comment]
Sorry should have added the design @kadams54 

https://cloud.githubusercontent.com/assets/9105999/14288312/01d7e9de-fb4e-11e5-9892-3ee3f0847ac9.png

**Comment by kadams54** [comment]
QA is :ok:, code is :+1:

File: jujugui/static/gui/src/app/components/user-profile/entity/entity.js (hunk 1 above)
Comment (kadams54) [inline jujugui/static/gui/src/app/components/user-profile/entity/entity.js]: I believe @huwshimi's data validation branch prevents a slash from being in the model name.

**Comment by hatched** [comment]
Thanks everyone! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**